<ui version="4.0" >
 <class>JoystickView</class>
 <widget class="QWidget" name="JoystickView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>218</height>
   </rect>
  </property>
  <property name="mouseTracking" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>Joystick Viewer</string>
  </property>
  <property name="windowIcon" >
   <iconset>images/joystick.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="infoTab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="locationLabel" >
             <property name="text" >
              <string>Location</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nameLabel" >
             <property name="text" >
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="driverVersionLabel" >
             <property name="text" >
              <string>Driver Version</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="buttonCountLabel" >
             <property name="text" >
              <string>Buttons</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="axisCountLabel" >
             <property name="text" >
              <string>Axes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="locationEdit" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameEdit" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="driverVersionEdit" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="buttonCountEdit" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="axisCountEdit" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="axesTab" >
      <attribute name="title" >
       <string>Axes</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="xAxislabel" >
             <property name="text" >
              <string>X-Axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="yAxisLabel" >
             <property name="text" >
              <string>Y-Axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="wheelAxisLabel" >
             <property name="text" >
              <string>Wheel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="xAxisEdit" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yAxisEdit" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="wheelAxisEdit" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="buttonTab" >
      <attribute name="title" >
       <string>Buttons</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTableWidget" name="buttonTable" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="lineWidth" >
          <number>1</number>
         </property>
         <property name="dragDropOverwriteMode" >
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text" >
           <string>State</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
